ocs_ci/deployment/deployment.py

```diff
@@ -418,10 +421,21 @@ def deploy_ocs_via_operator(self):
             run_cmd(f"oc create -f {self.CUSTOM_STORAGE_CLASS_PATH}")
 
         # creating StorageCluster
-        if self.platform == constants.IBM_POWER_PLATFORM:
-            cluster_data = templating.load_yaml(constants.IBM_STORAGE_CLUSTER_YAML)
+        if config.ENV_DATA.get("enable_flexible_scaling"):
+            if ocs_version < 4.7:
+                error_message = "Flexible scaling can be enabled only on OCS >= 4.7!"
+                logger.error(error_message)
+                raise UnsupportedFeatureError(error_message)
+            else:
+                logger.info("Enabling flexible scaling!")
+                cluster_data = templating.load_yaml(
```

Comment:
You should not have separate file for storageCluster CR for such eaasy change of one value, see my other comment. I think you should just change this value from already loaded cluster_data, so move this code somewhere below where we already have data loaded and update value with:
```python
cluster_data['spec']['flexibleScaling'] =  True
```
And that's it I think no need for whole storageCluster CR as it's the same as our default one.

Reply:
I created a new file because the storageDeviceSet.count and  storageDeviceSet.replica also change for flexible scaling and I thought it would be cleaner.
Would you still prefer that I can update the code instead?

Reply:
I prefer to update those values in the code as well. @clacroix12 @shylesh @vavuthu  do you agree here?

Reply:
If it is minimal changes for flexible scaling, I would like to update those values in code.

Reply:
Other similar changes to the storageCluster (and other) CRs are done in code so my vote is to remain consistent.